### DISCUSSION TITLE
KEDA operator and metric server Memory Profiling

### DISCUSSION THREAD

**Discussion by kevinmat** [kedacore/keda#1550] [Q&A / Need Help]
Hello,
I wanted to check the memory graphs of KEDA operator , is there any metrics which can be exported to Prometheus so that I view the graphs on Grafana

I have setup KEDA on kubernetes. I tried logging into the KEDA pod but was unable to do so

kubectl exec -it keda-operator-6d77668775-mdfj4 -n kedans

Thanks in advance,
Kevin

**Comment by zroubalik** [comment]
You can check Promethes Metrics exposed by KEDA: https://keda.sh/docs/2.1/operate/prometheus/

**Reply by kevinmat** [reply]
@zroubalik  Thanks for the reply 
The data on the port contains all the metrics mentioned in the URL which you gave
I was looking for the statistics like Memory , CPU , Open Files etc.. of the KEDA operator and KEDA metrics server

**Reply by zroubalik** [reply]
Please take a look on this PR: https://github.com/kedacore/charts/pull/118 looking forward for a feedback
